includes/migration-guide/retargeting/core/changes-path-normalization.md

```diff
@@ -2,7 +2,7 @@
 |---|---|
-|Details|Starting with apps that target the .NET Framework 4.6.2, the way in which the runtime normalizes paths has changed.Normalizing a path involves modifying the string that identifies a path or file so that it conforms to a valid path on the target operating system. Normalization typically involves:<ul><li>Canonicalizing component and directory separators.</li><li>Applying the current directory to a relative path.</li><li>Evaluating the relative directory (.) or the parent directory (..) in a path.</li><li>Trimming specified characters.</li></ul>Starting with apps that target the .NET Framework 4.6.2, the following changes in path normalization are enabled by default:<ul><li>The runtime defers to the operating system's [GetFullPathName](https://msdn.microsoft.com/library/windows/desktop/aa364963(v=vs.85).aspx) function to normalize paths.</li><li>Normalization no longer involves trimming the end of directory segments (such as a space at the end of a directory name).</li><li>Support for device path syntax in full trust, including `\\.\` and, for file I/O APIs in mscorlib.dll, `\\?\`.</li><li>The runtime does not validate device syntax paths.</li><li>The use of device syntax to access alternate data streams is supported.</li></ul>These changes improve performance while allowing methods to access previously inaccessible paths. Apps that target the .NET Framework 4.6.1 and earlier versions but are running under the .NET Framework 4.6.2 or later are unaffected by this change.|
+|Details|Starting with apps that target the .NET Framework 4.6.2, the way in which the runtime normalizes paths has changed.Normalizing a path involves modifying the string that identifies a path or file so that it conforms to a valid path on the target operating system. Normalization typically involves:<ul><li>Canonicalizing component and directory separators.</li><li>Applying the current directory to a relative path.</li><li>Evaluating the relative directory (.) or the parent directory (..) in a path.</li><li>Trimming specified characters.</li></ul>Starting with apps that target the .NET Framework 4.6.2, the following changes in path normalization are enabled by default:<ul><li>The runtime defers to the operating system's [GetFullPathName](/windows/desktop/api/fileapi/nf-fileapi-getfullpathnamea) function to normalize paths.</li><li>Normalization no longer involves trimming the end of directory segments (such as a space at the end of a directory name).</li><li>Support for device path syntax in full trust, including `\\.\` and, for file I/O APIs in mscorlib.dll, `\\?\`.</li><li>The runtime does not validate device syntax paths.</li><li>The use of device syntax to access alternate data streams is supported.</li></ul>These changes improve performance while allowing methods to access previously inaccessible paths. Apps that target the .NET Framework 4.6.1 and earlier versions but are running under the .NET Framework 4.6.2 or later are unaffected by this change.|
```

Comment:
@rpetrusha FYI this impacts the app-compat includes. Since this content from the folder includes/migration-guide is also used somewhere else, let's add the domain on these 

```suggestion
|Details|Starting with apps that target the .NET Framework 4.6.2, the way in which the runtime normalizes paths has changed.Normalizing a path involves modifying the string that identifies a path or file so that it conforms to a valid path on the target operating system. Normalization typically involves:<ul><li>Canonicalizing component and directory separators.</li><li>Applying the current directory to a relative path.</li><li>Evaluating the relative directory (.) or the parent directory (..) in a path.</li><li>Trimming specified characters.</li></ul>Starting with apps that target the .NET Framework 4.6.2, the following changes in path normalization are enabled by default:<ul><li>The runtime defers to the operating system's [GetFullPathName](https://docs.microsoft.com/windows/desktop/api/fileapi/nf-fileapi-getfullpathnamea) function to normalize paths.</li><li>Normalization no longer involves trimming the end of directory segments (such as a space at the end of a directory name).</li><li>Support for device path syntax in full trust, including `\\.\` and, for file I/O APIs in mscorlib.dll, `\\?\`.</li><li>The runtime does not validate device syntax paths.</li><li>The use of device syntax to access alternate data streams is supported.</li></ul>These changes improve performance while allowing methods to access previously inaccessible paths. Apps that target the .NET Framework 4.6.1 and earlier versions but are running under the .NET Framework 4.6.2 or later are unaffected by this change.|
```